libs/net/dal/Migrations/1.0.134/Up/PostUp/00-ReportTemplate.sql

```diff
@@ -0,0 +1,293 @@
+DO $$
+BEGIN
+
+UPDATE public.report_template SET
```

Comment:
Cleaned up formatting of code.
Provide option to not include front page images in ToC.